test/system/libs/helpers.bash

```diff
@@ -2,15 +2,18 @@
 
 load 'libs/bats-support/load'
 
+# Helpful globals
+readonly PROJECT_DIR=${PWD}
```

Comment:
I wonder if going forward a better location to use for out temp files/image cache would be somewhere else than `$PWD` or `tmpfs` (which [cannot](https://github.com/containers/podman/issues/10693#issuecomment-863007516) be used for Podman store). The use of `$PWD` could be problematic when having the tests installed in a system-location (e.g., being installed via a a package manager into `/usr/share/`).

Maybe we could use a location in `XDG_RUNTIME_DIR`? Of course, a default has to be set in case the tests are run as root or a user without the variable set.

Reply:
It is a good point, but I think this would be done in a different issue/pr. We need to think about the pros/cons of different places to store temporary data and the possible use cases.

Reply:
Fair enough. Let's stick to what you have then.